Source/CloudyPanelPlugin/Private/CloudyPanelPlugin.cpp

```diff
@@ -266,77 +257,45 @@ void CCloudyPanelPluginModule::StreamFrameToClient() {
 	// use VideoPipe (class variable) to pass frames to encoder
 	uint32 *PixelBuffer;
 	FColor Pixel;
-	for (int i = 0; i < NumberOfPlayers; i++) {
-		PixelBuffer = new uint32[sizeX * sizeY * PIXEL_SIZE];
-		//UE_LOG(ModuleLog, Warning, TEXT("FrameIndex %d"), FrameIndex);
-		for (int j = 0; j < FrameBufferList[i].Num(); j++) {
+	PixelBuffer = new uint32[sizeX * sizeY * PIXEL_SIZE];
+
+	for (int i = 0; i < NumberOfPlayers; i++) {	
+		int FrameSize = FrameBufferList[i].Num();
+		for (int j = 0; j < FrameSize; j++) {
```

Comment:
Try using ++j instead. The difference might be significant due to the FrameSize.